dom0/sd-clean-all.sls

```diff
@@ -21,6 +21,7 @@ remove-dom0-sdw-config-files:
       - /etc/qubes-rpc/policy/securedrop.Log
       - /etc/qubes-rpc/policy/securedrop.Proxy
       - /home/{{ gui_user }}/Desktop/securedrop-launcher.desktop
+      - /home/{{ gui_user }}/.securedrop_launcher/sdw-update-flag
```

Comment:
We should at least remove the other flags as well (the current, working flags) and consider deleting the entire folder (which include the logs)

Reply:
I was wondering about that; I thought you maybe intentionally omitted the folder. But I agree, it should clean up everything.

Reply:
Done and tested. If you prefer to preserve the logs we can list individual files instead, but it would feel a bit wrong to me to leave this directory hanging around.